onnxruntime/core/providers/op_kernel_type_control_overrides.inc

```diff
@@ -0,0 +1,10 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+namespace onnxruntime {
+namespace op_kernel_type_control {
+
+// specify allowed types here
+
```

Comment:
Would be good to have one example entry for a global list, and a per operator list.

May also be good to add tags for begin/end insertion points so the reduction script knows where to put things. Usage would be it replaces anything between the begin and end markers with new content (allows it to be run multiple times), and leaves anything outside the markers alone.